From 133ac64836d0c82c558dc799b66cb52442aa3180 Mon Sep 17 00:00:00 2001
From: Alex Caithness <acaithness@cclgroupltd.com>
Date: Wed, 13 Jul 2022 15:57:14 +0100
Subject: [PATCH 08/19] Remove files found in the interface log

---
 aleapp.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/aleapp.py b/aleapp.py
index 72d7d16..0434da2 100644
--- a/aleapp.py
+++ b/aleapp.py
@@ -148,7 +148,6 @@ def crunch_artifacts(
                 for pathh in found:
                     if pathh.startswith('\\\\?\\'):
                         pathh = pathh[4:]
-                    logfunc(f'Files for {artifact_search_regex} located at {pathh}')
                     log.write(f'Files for {artifact_search_regex} located at {pathh}<br><br>')
                 files_found.extend(found)
         if files_found:
-- 
2.34.1

